bin/pycbc_live

```diff
@@ -505,6 +508,9 @@ parser.add_argument('--enable-profiling', type=int, metavar='RANK',
                          ' with given rank at the end of program execution')
 
 parser.add_argument('--enable-background-estimation', default=False, action='store_true')
+parser.add_argument('--conservative-sngl-ifar-est', default=True, action='store_true',
+                    help='Use conservative trigger fit values in estimation of '
+                         'single trigger ifars. Default=True')
```

Comment:
How does a `store_true` option work if it defaults to `True`? How do you set it to `False`?

Reply:
Well that was a thinko and didn't make sense - fortunately I'm not using this option any more (forgot to remove it) so it doesn't matter